flixel/util/FlxTimer.hx

```diff
@@ -273,4 +273,14 @@ class FlxTimerManager extends FlxBasic
 	{
 		FlxArrayUtil.clearArray(_timers);
 	}
-}
\ No newline at end of file
+	
+	/**
+	 * Calls the onComplete function of all timers, then removes them from the timer manager.
+	 */
+	public function completeAll():Void
+	{
+		for (timer in _timers)
+			timer.onComplete(timer);
```

Comment:
OK, fair enough.

Unfortunately, I have trouble getting it to work the other way:

```haxe
	public function completeAll():Void
	{
		for (timer in _timers)
			if (timer.loops == 1)
				timer.update(timer.timeLeft);
	}
```
...this somehow only fires about half my timers, whereas the original approach was working fine for me, somehow.